lighthouse-core/audits/first-meaningful-paint.js

```diff
@@ -40,6 +40,7 @@ class FirstMeaningfulPaint extends Audit {
       name: 'first-meaningful-paint',
       description: 'First meaningful paint',
       optimalValue: SCORING_POINT_OF_DIMINISHING_RETURNS.toLocaleString() + 'ms',
+      helpText: 'Load time is a key factor in how users perceive the performance of your app. A low first meaningful paint means that users perceive your app as loading fast. <a href="https://developers.google.com/web/tools/lighthouse/audits/first-meaningful-paint" target="lighthouse" rel="noreferrer noopener">Learn more</a>.',
```

Comment:
Think you can shorten to:

"A fast first meaningful paint is a key factor in how users perceive the loading performance of your app."